TASVideos/Pages/Search/Index.cshtml.cs

```diff
@@ -73,7 +73,7 @@ public async Task<IActionResult> OnGet()
 
 			GameResults = await _db.Games
 				.Where(g => EF.Functions.ToTsVector(g.DisplayName + " || " + g.Aliases + " || " + g.Abbreviation).Matches(EF.Functions.WebSearchToTsQuery(SearchTerms)))
-				.OrderBy(g => g.DisplayName)
+				.OrderBy(g => g.GameGenres.Select(gg => gg.GenreId).Contains(14)).ThenBy(g => g.DisplayName)
```

Comment:
Should be better to use a named constant, or even pull human readable meaning of the value from genre strings, if it's there.

Reply:
Better now? I used the Display Name property instead of Id